src/identity-reuse/tests/calculate-vot.test.ts

```diff
@@ -16,11 +27,39 @@ describe("calculate-vot", () => {
     ["P2", ["P2", "P3"], "P2"],
     ["P2", ["P3", "P2"], "P2"],
     ["P0", ["P3"], "P2"],
+  ])("should return %s, for vtr %s and unsigned vot %s when max_vot not present", (expected, vtr, unsignedVot) => {
+    jest.mock("../../commons/logger");
+
+    const jwt: StoredIdentityJWT = {} as StoredIdentityJWT;
+    const returnedVot = calculateVot(jwt, unsignedVot, vtr);
+
+    expect(mockedLogger.warn).toHaveBeenCalledWith("Max VOT not in VC. Using unsigned VOT");
```

Comment:
Oh was wondering about this, so the key assertion here is in the line below - right? I guess the practice of making assertions on console log messages feels a bit funny to me, could we possibly remove it, only relying on on more intention side effects of the program e.g. a file gets created, or in this case, a value is returned. In this case a function return value feels like a sensible thing to test, so the below assertion is good. But intercepting log messages feels brittle, and like it's testing the wrong thing.